packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -138,6 +138,57 @@ export default class Reachability extends EventsScope {
     }
   }
 
+  /**
+   * Checks if the given subnet is reachable
+   * @param {string} mediaServerIp - media server ip
+   * @returns {boolean} true if reachable, false otherwise
+   * @public
+   * @memberof Reachability
+   */
+  public isSubnetReachable(mediaServerIp?: string): boolean | null {
+    if (!mediaServerIp) {
+      LoggerProxy.logger.error(`Reachability:index#isSubnetReachable --> mediaServerIp is null`);
+
+      return null;
+    }
+
+    const subnetFirstOctet = mediaServerIp.split('.')[0];
+
+    LoggerProxy.logger.info(
+      `Reachability:index#isSubnetReachable --> Looking for subnet: ${subnetFirstOctet}.X.X.X`
+    );
+
+    const reachedClusters = Object.values(this.clusterReachability).reduce((acc, cluster) => {
+      const reachedSubnetsArray = Array.from(cluster.reachedSubnets);
+
+      let logMessage = `Reachability:index#isSubnetReachable --> Cluster ${cluster.name} reached [`;
+      for (let i = 0; i < reachedSubnetsArray.length; i += 1) {
+        const subnet = reachedSubnetsArray[i];
+        const reachedSubnetFirstOctet = subnet.split('.')[0];
+
+        if (subnetFirstOctet === reachedSubnetFirstOctet) {
+          acc.add(cluster.name);
+        }
+
+        logMessage += `${subnet}`;
+        if (i < reachedSubnetsArray.length - 1) {
+          logMessage += ',';
+        }
+      }
+      logMessage += `]`;
+
+      LoggerProxy.logger.info(logMessage);
```

Comment:
(2) and then use it here:
```suggestion
      LoggerProxy.logger.info(
  `Reachability:index#isSubnetReachable --> Cluster ${cluster.name} reached [${logMessage}]`
);
```